backend/src/main/java/eu/bbmri_eric/negotiator/common/PdfMerger.java

```diff
@@ -0,0 +1,57 @@
+package eu.bbmri_eric.negotiator.common;
+
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.util.List;
+import lombok.extern.apachecommons.CommonsLog;
+import org.apache.pdfbox.Loader;
+import org.apache.pdfbox.multipdf.PDFMergerUtility;
+import org.apache.pdfbox.pdmodel.PDDocument;
+import org.apache.pdfbox.pdmodel.PDPage;
+
+/** Utility class for merging multiple PDF documents into a single PDF. */
+@CommonsLog
+public class PdfMerger {
```

Comment:
Why make such a narrow utility class public?

Reply:
Moved to the `pdf` package amd changed to default class modifier